Sources/SwiftDocC/Model/Rendering/Variants/VariantCollection+Symbol.swift

```diff
@@ -84,6 +84,46 @@ public extension VariantCollection {
         self.init(defaultValue: defaultValue, variants: variants)
     }
     
+    /// Creates a variant collection from a given set of variant traits.
+    ///
+    /// If there are no variants for the given traits, the transform closure is called
+    /// with a `nil` value.
```

Comment:
Is this accurate? It seems like if there are no variants, the initializer returns `nil`.

Reply:
Ah no- I think that's out-of-date. Good catch, thanks!